doc/source/serve/deployment-graph/deployment_graph_intro.md

```diff
@@ -0,0 +1,219 @@
+(serve-intro-to-deployment-graphs)=
+
+# Intro to Deployment Graphs
+
+```{note}
+Note: This feature is in Alpha, so APIs are subject to change.
+```
+
+This section should help you:
+
+* compose your Ray Serve deployments together with the **Deployment Graph** API
+* serve your applications that use multi-model inference, ensemble models, ML model composition, or mixed business logic/model inference workloads
+* independently scale each of your ML models and business logic steps
+
+Ray Serve's **Deployment Graph** API lets you compose your deployments together by describing how to route a request through your deployments. This is particularly useful if you're using ML model composition or mixing business logic and model inference in your application. You can encapsulate each of your models and each of your business logic steps in independent deployments. Then, you can chain these deployments together in a deployment graph.
```

Comment:
Hmm, yeah this is a bit tricky. Even single-node deployments are using the deployment graph API (such as `bind` and `serve.run`), so users should always be using this API. The rest of the documentation doesn't really dwell on this.

It might be nice to acknowledge this here, though. Do you have any suggestions on how we do that?

Reply:
In that case I think we can just leave it as is!

Reply:
I wonder if you want to have subheadings for the landing page that reads:
What are Deployments Graphs?
Why and when to use Deployment Graphs?
How to use them?

The developer is looking for this quick information what, why or when, and how? I have suggested that format Ray AIR. The how bit becomes your guickstart.